Plugins/OceanPlugin/Source/OceanPlugin/OceanPlugin.Build.cs

```diff
@@ -35,7 +35,7 @@ private string ThirdPartyPath
 
     public OceanPlugin(ReadOnlyTargetRules Target) : base(Target)
     {
-        PrivateIncludePaths.AddRange(new string[] { "OceanPlugin/Private" });
+        PrivateIncludePaths.AddRange(new string[] { Path.Combine(ModuleDirectory, "Private") });
```

Comment:
Is this still compatible as a standalone plugin? Without making this an engine module?